frontend/.eslintrc.js

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Can we have a single config file at root level and import and overwrite rules in sub projects accordingly

Reply:
yeah once I've migrated all the repos it will be great to make any duplicate configs global that we can